### PR TITLE
add resources requests to prow components.

### PR DESCRIPTION
**Pull request by droslean**

### CODE REVIEW

File: cluster/ci/config/prow/openshift/artifact-uploader.yaml [hunk 1]
```diff
@@ -39,6 +39,15 @@ items:
           - name: gcs-credentials
             mountPath: /secrets/gcs
             readOnly: true
+          resources:
+            requests:
+              memory: "2Pi"
+              cpu: "100m"
+              ephemeral-storage: "30Mi"
+            limits:
+              memory: "4Pi"
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/openshift/artifact-uploader.yaml]: What is `Pi`?

File: cluster/ci/config/prow/openshift/cherrypick.yaml [hunk 2]
```diff
@@ -50,6 +50,15 @@ items:
             readOnly: true
           - name: tmp
             mountPath: /tmp
+          resources:
+            requests:
+              memory: "3Pi"
+              cpu: "250m"
+              ephemeral-storage: "2Mi"
+            limits:
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/openshift/cherrypick.yaml]: I don't think we want limits, just requests.

File: cluster/ci/config/prow/openshift/artifact-uploader.yaml (hunk 1 above)
Comment (droslean) [reply cluster/ci/config/prow/openshift/artifact-uploader.yaml]: According to this [doc](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory), 

> Limits and requests for memory are measured in bytes. You can express memory as a plain integer or as a fixed-point integer using one of these suffixes: E, P, T, G, M, K. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki. F

File: cluster/ci/config/prow/openshift/cherrypick.yaml (hunk 2 above)
Comment (droslean) [reply cluster/ci/config/prow/openshift/cherrypick.yaml]: ok let me remove them

File: cluster/ci/config/prow/openshift/artifact-uploader.yaml (hunk 1 above)
Comment (stevekuznetsov) [reply cluster/ci/config/prow/openshift/artifact-uploader.yaml]: So are you specifying petabytes of memory?

Comment (droslean) [reply cluster/ci/config/prow/openshift/artifact-uploader.yaml]: I tried to adjust the request regarding the grafana dashboard. It seems the I messed it up with the conversion. Let me fix that.

File: cluster/ci/config/prow/openshift/jenkins_operator.yaml [hunk 3]
```diff
@@ -358,6 +368,15 @@ items:
           - name: job-config-43
             mountPath: /etc/job-config/4.3
             readOnly: true
+          resources:
+            requests:
+              memory: "500Mi"
+              cpu: "100m"
+              ephemeral-storage: "10Mi"
+            limits:
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/openshift/jenkins_operator.yaml]: oops

**Pull request merged**